.github/workflows/update_disabled_tests.yml

```diff
@@ -6,6 +6,9 @@ on:
     - cron: "*/15 * * * *"
   # Have the ability to trigger this job manually through the API
   workflow_dispatch:
+  pull_request:
+    paths:
+      - .github/scripts/update_disabled_issues.py
 
 jobs:
   update-disabled-tests:
```

Comment:
I could also update the job name here and rename the workflow (I wonder if that breaks anything)